### PR TITLE
bazel_6: 6.1.2 -> 6.2.0

### PR DESCRIPTION
**Pull request by divanorama**
###### Description of changes

https://github.com/bazelbuild/bazel/releases/tag/6.2.0

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by uri-canva — APPROVED** [review]
Nice! Is it draft because you want to merge this after https://github.com/NixOS/nixpkgs/pull/231437? Or is there something else that makes it not ready to merge? Did you check if you need to update the hashes of `tensorflow-probability`?

**Comment by divanorama** [comment]
> Nice! Is it draft because you want to merge this after #231437? Or is there something else that makes it not ready to merge? Did you check if you need to update the hashes of `tensorflow-probability`?

Yes, want to get the darwin fix in first, then also to look for hashes to update

**Comment by divanorama** [comment]
@ofborg build python3Packages.tensorflow-probability

**Comment by divanorama** [comment]
@ofborg build envoy
@ofborg build python3Packages.tensorflow-probability

**Comment by divanorama** [comment]
@ofborg build envoy
@ofborg build python3Packages.tensorflow-probability

File: pkgs/servers/http/envoy/default.nix [hunk 1]
```diff
@@ -80,8 +80,8 @@ buildBazelPackage rec {
 
   fetchAttrs = {
     sha256 = {
-      x86_64-linux = "sha256-jx8RavJKlBtZQtZf4GrUkOlhM6qI4LO5lK3HRgn1vzE=";
-      aarch64-linux = "sha256-toEPrGVh61sHsVbhsideMmnX8uXIN+2x8LuZpczTEdw=";
+      x86_64-linux = "sha256-mw3k2r4heoAcBdcc7uYdnotUBrF1nM5Vmqbay+2DkjI=";
+      aarch64-linux = "sha256-rR6EFEFh01P9T42g9OlM15xP7V4ya4TQJ2bjUWjSaxU=";
```
Comment (divanorama) [inline pkgs/servers/http/envoy/default.nix:84]:
```suggestion
      aarch64-linux = "sha256-2gSxzm7SXvrGEgwZnp5KdEpbV/+zdosf8Z5lrkK3QiI=";
```

**Comment by divanorama** [comment]
@ofborg build envoy
@ofborg build python3Packages.tensorflow-probability

**Review by uri-canva — APPROVED** [review]

**Pull request merged**